recipes-core/initrdscripts/initramfs-framework/aosupdate

```diff
@@ -19,11 +19,28 @@ UPDATE_IMAGE=""
 UPDATE_DIR="${bootparam_aosupdate_disk:+${UPDATE_MOUNT}/}${bootparam_aosupdate_path:-update_rootfs}"
 UPDATE_EXT="*.squashfs"
 
+log() {
```

Comment:
Better to use existin function for log output. I believe there is another solution to redirect it to systems journal. 

See:

poky/meta/recipes-core/initrdscripts/initramfs-framework/init
meta-aos-vm/meta-aos-vm-common/recipes-bsp/grub/grub-config.bb

I guess `console` or some other kernel param should solve the issue.